src/pe_reports/report_generator.py

```diff
@@ -1,14 +1,30 @@
-"""A tool for creating Posture & Exposure reports."""
+"""A tool for creating Posture & Exposure reports.
+
+Usage:
+    pe-reports REPORT_DATE  DATA_DIRECTORY OUTPUT_DIRECTORY
+
+Options:
+    -h --help                       Show this message.
+    REPORT_DATE                     Date of the report, format YYYY-MM-DD
```

Comment:
Add period to be consistent with other lines
```suggestion
    REPORT_DATE                     Date of the report, format YYYY-MM-DD.
```

Reply:
Resolved in 1299aba